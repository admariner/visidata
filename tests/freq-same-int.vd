sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/benchmark.csv	o	
benchmark	Date		type-date		@	
benchmark	Date		key-col		!	
benchmark			select-rows		gs	
benchmark	Quantity	キ2018-07-03	setcol-input	3	ge	
benchmark			unselect-rows		gu	
benchmark	Quantity		type-int		#	
benchmark	Quantity		freq-col		F	
benchmark	Quantity	キ2018-07-03	edit-cell	2	e	
benchmark	Quantity		freq-col		F	
benchmark_Quantity_freq	histogram		hide-col		-	Hide current column
