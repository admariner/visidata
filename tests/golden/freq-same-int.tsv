Quantity	count	percent
2	1	1.96
3	50	98.04
